index.js

```diff
@@ -39,9 +65,9 @@ function cardHtml(quest) {
         }
         ret += '<ul style="margin: 0 0 1ex 2em; padding: 0px; list-style: none; text-indent: -1em">';
         for (var i = 0; i < quest.flavor.length; ++i) {
-            ret += '<li style="margin: 5px 0"><img src="http://xavid.us/' + quest.flavor[i][0] + '.png" width="20" style="vertical-align: middle">';
+            ret += '<li style="margin: 5px 0"><img src="img/' + quest.flavor[i][0] + '.png" width="20" style="vertical-align: middle">';
             if (quest.flavor[i][1]) {
-                ret += '<img src="http://xavid.us/' + quest.flavor[i][1] + '.png" width="20" style="vertical-align: middle">';
+                ret += '<img src="img/' + quest.flavor[i][1] + '.png" width="20" style="vertical-align: middle">';
```

Comment:
So, the original reason I used absolute paths for images is so you can copy this HTML block into a Dreamwidth post (or whatever your favorite blog that allows HTML is) and it just works. So I'd prefer to at least keep an option that maintains that capability.

Reply:
Just pushed a new change which leaves in the absolute urls but makes it easier to adjust during local development.

The images will still be located in a subfolder--if you prefer keeping it at the root level of the website, I can move them all up to be at the same level as the HTML/JS files.